src/cmds/qrerun.c

```diff
@@ -132,7 +132,7 @@ main(int argc, char **argv, char **envp) /* qrerun */
 		}
 
 		stat = pbs_rerunjob(connect, job_id_out, extra);
-		if (stat && (pbs_errno != PBSE_UNKJOBID)) {
+		if ((stat && (pbs_errno != PBSE_UNKJOBID)) || (pbs_geterrmsg(connect))) {
```

Comment:
Now that we throw a different exit value, there is a need to add this to external design change.
Please make the function header doxygen compliant and if since we don't use the word timeout, I suggest to change the function name as well..